File: app/src/main/java/com/iita/akilimo/mappers/ComputedResponse.java
package com.iita.akilimo.mappers;


public class ComputedResponse {
    private String computedTitle;
    private String computedRecommendation;

    public ComputedResponse createObject(String title, String body) {
        this.setComputedTitle(title);
        this.setComputedRecommendation(body);
        return this;
    }

    public String getComputedTitle() {
        return computedTitle;
    }

    public void setComputedTitle(String computedTitle) {
        this.computedTitle = computedTitle;
    }

    public String getComputedRecommendation() {
        return computedRecommendation;
    }

    public void setComputedRecommendation(String computedRecommendation) {
        this.computedRecommendation = computedRecommendation;
    }
}

File: app/src/main/java/com/akilimo/mobile/mappers/ComputedResponse.kt
package com.akilimo.mobile.mappers

class ComputedResponse {
    var computedTitle: String? = null
    var computedRecommendation: String? = null

    fun createObject(title: String?, body: String?): ComputedResponse {
        computedTitle = title
        computedRecommendation = body
        return this
    }

}
